### PR TITLE
Task 5

### PR DESCRIPTION
**Pull request by SkorykV**
Example of Parser logs:
image
**Get signed url endpoint:** GET - https://asb3mmmcsa.execute-api.eu-west-1.amazonaws.com/dev/import
Link to website: https://de9bltgxbvako.cloudfront.net/
Link to FE MR: https://github.com/SkorykV/nodejs-aws-fe/pull/4/files
**Main scope**
- [x]  File serverless.yml contains configuration for importProductsFile function
- [x] The importProductsFile lambda function returns a correct response which can be used to upload a file into the S3 bucket
- [x] Frontend application is integrated with importProductsFile lambda
- [x] The importFileParser lambda function is implemented and serverless.yml contains configuration for the lambda

**Additional (optional) tasks:**
- [x] async/await is used in lambda functions
- [x] importProductsFile lambda is covered by unit tests (aws-sdk-mock can be used to mock S3 methods - https://www.npmjs.com/package/aws-sdk-mock)
- [x] At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)

Expected score: 8/8

### CODE REVIEW

File: import-service/src/handlers/import-file-parser.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      const bucket = record.s3.bucket.name;
+      const key = decodeURIComponent(record.s3.object.key.replace(/\+/g, ' '));
+      const parsedFilesFolder = 'parsed';
+
+      await parseService.process(bucket, key, parsedFilesFolder);
+    })(),
+  );
+
+  await Promise.all(promises);
+  console.log('importFileParser finished');
```
Comment (Ivanko5417) [inline import-service/src/handlers/import-file-parser.js:18]: this handler should be a correct HTTP response code according to the Task 5.2

**Review by loginov-rocks — APPROVED** [review]
Nice work!

**Review by byslavik — APPROVED** [review]
Looks great!

**Review by Ivanko5417 — CHANGES_REQUESTED** [review]

File: import-service/src/handlers/import-file-parser.js (hunk 1 above)
Comment (SkorykV) [reply import-service/src/handlers/import-file-parser.js:18]: that task description is wrong. This lambda isn't called via HTTP. It doesn`t need any response. You could check this example https://docs.aws.amazon.com/lambda/latest/dg/with-s3-example.html.

**Pull request merged**